detectionExample/ObjectWrapper.py

```diff
@@ -15,36 +15,42 @@ def __init__(self, bbox):
         self.name = bbox.name
 
 class ObjectWrapper():
+    mvnc.SetGlobalOption(mvnc.GlobalOption.LOG_LEVEL, 2)
+    devices = mvnc.EnumerateDevices()
+    devNum = len(devices)
+    if len(devices) == 0:
+        print('No MVNC devices found')
+        quit()
+    devHandle = []
+    graphHandle = []
     def __init__(self, graphfile):
         select = 1
         self.detector = YoloDetector(select)
-        mvnc.SetGlobalOption(mvnc.GlobalOption.LOG_LEVEL, 2)
-        devices = mvnc.EnumerateDevices()
-        if len(devices) == 0:
-            print('No MVNC devices found')
-            quit()
-        self.device = mvnc.Device(devices[0])
-        self.device.OpenDevice()
-        opt = self.device.GetDeviceOption(mvnc.DeviceOption.OPTIMISATION_LIST)
-        # load blob
-        with open(graphfile, mode='rb') as f:
-            blob = f.read()
-        self.graph = self.device.AllocateGraph(blob)
-        self.graph.SetGraphOption(mvnc.GraphOption.ITERATIONS, 1)
-        iterations = self.graph.GetGraphOption(mvnc.GraphOption.ITERATIONS)
-
-        self.dim = (416,416)
-        self.blockwd = 12
-        self.wh = self.blockwd*self.blockwd
-        self.targetBlockwd = 12
-        self.classes = 20
-        self.threshold = 0.2
-        self.nms = 0.4
+        
+        for i in range(ObjectWrapper.devNum):
+            ObjectWrapper.devHandle.append(mvnc.Device(ObjectWrapper.devices[i]))
+            ObjectWrapper.devHandle[i].OpenDevice()
+            opt = ObjectWrapper.devHandle[i].GetDeviceOption(mvnc.DeviceOption.OPTIMISATION_LIST)
+            # load blob
+            with open(graphfile, mode='rb') as f:
+                blob = f.read()
+            ObjectWrapper.graphHandle.append(ObjectWrapper.devHandle[i].AllocateGraph(blob))
+            ObjectWrapper.graphHandle[i].SetGraphOption(mvnc.GraphOption.ITERATIONS, 1)
+            iterations = ObjectWrapper.graphHandle[i].GetGraphOption(mvnc.GraphOption.ITERATIONS)
+
+            self.dim = (416,416)
+            self.blockwd = 13
```

Comment:
Sorry, I have forgot to restore these 3 parameters: blockwd, targetBlockwd and classes.